### ISSUE TITLE
Cannot find module

### ISSUE THREAD

**Issue by dmoKruso** [aaronhayes/react-use-hubspot-form#37]
- **I'm submitting a ...**
  [x] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**

Cannot find module '@aaronhayes/react-use-hubspot-form' or its corresponding type declarations.ts(2307)

**Comment by aaronhayes** [comment]
Did you install the package?

**Comment by dmoKruso** [comment]
Yes, ofcourse. It only started to give error after packages updates as react and ts update. 
Adding // @ts-ignore it solves the warning.

**Comment by aaronhayes** [comment]
Closing this as typescript config issue
